src/pages/library/DataBrowserDetails.js

```diff
@@ -109,18 +109,18 @@ const LegacyAttributesComponent = ({ dataObj }) => {
 }
 
 const MainContent = ({ dataObj }) => {
+  const accessURL = dataObj['dcat:accessURL']
+  const workspaceName = accessURL?.includes('/#workspaces/') && accessURL.substring(accessURL.lastIndexOf('/') + 1)
   return div({ style: { ...styles.content, width: '100%', marginTop: 0 } }, [
     h1({ style: { lineHeight: '1.5em' } }, [dataObj['dct:title']]),
-    dataObj.dataSource && div({ style: { marginBottom: '1rem' } }, [
-      `This data is from ${dataObj.dataSource.storageSystemName}:`,
-      h(ButtonSecondary, {
-        style: { fontSize: 16, textTransform: 'none', height: 'unset', display: 'block' },
-        onClick: () => {
-          // TODO: fill out this link
-        }
+    workspaceName && div({ style: { marginBottom: '1rem' } }, [
+      `This data is from the Terra workspace:`,
```

Comment:
```suggestion
      'This data is from the Terra workspace:',
```

since we're not doing any interpolation. Backticks in JS function as `s"…"` in Scala :)